### PR TITLE
Automate Maven Deployment

### PR DESCRIPTION
**Pull request by mhamilton723**

### CODE REVIEW

**Comment by mhamilton723** [comment]
https://jenkins.cntk.selfhost.corp.microsoft.com/job/CNTK-Build-And-Test-Workflow/17314/

File: bindings/java/Swig/pom.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.microsoft.cntk</groupId>
+    <artifactId>cntk</artifactId>
+    <version>2.4</version>
```
Comment (manikjindal) [inline bindings/java/Swig/pom.xml]: What is the impact of this version? Can it be a dynamic string? Version 2.4 won't comply with our nightly builds. Our nighlty builds has format 2.5-rc0.dev{yyyyMMdd}. You can check how I added dynamic version support in Python WHL creation (bindings/python/setup.py)

File: bindings/java/Swig/pom.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Manik Jindal</name>
+            <email>manikj@microsoft.com</email>
+            <organization>Microsoft</organization>
+            <organizationUrl>microsoft.com</organizationUrl>
+        </developer>
+        <developer>
```
[the remark this answers is not in the document]
Comment (mhamilton723) [reply bindings/java/Swig/pom.xml]: Yes I can remove him

File: bindings/java/Swig/pom.xml (hunk 1 above)
Comment (mhamilton723) [reply bindings/java/Swig/pom.xml]: unfortunately its xml so there is no easy way to add dynamic support from within the xml
however this only gets used when you actually publish the jar and is not used in the build process

Comment (manikjindal) [reply bindings/java/Swig/pom.xml]: Great then.

**Pull request merged**